### ISSUE TITLE
Crash on runJvMFirstThread . Propably related to Touchbar Mac

### ISSUE THREAD

**Issue by emign** [korlibs/korge#48]
Building and running with -XStartOnFirstThread parameter results in a crash on a 2018 Touchbar Macbook Pro. Code works in Windows VM and other Macs. 
```
------------------------------

Build time:   2019-10-18 00:28:36 UTC
Revision:     bd168bbf5d152c479186a897f2cea494b7875d13

Kotlin:       1.3.41
Groovy:       2.5.4
Ant:          Apache Ant(TM) version 1.9.14 compiled on March 12 2019
JVM:          13.0.1 (Oracle Corporation 13.0.1+9)
OS:           Mac OS X 10.15.1 x86_64

```

```
> Task :runJvmFirstThread FAILED
-XstartOnFirstThread not supported via Jogl, switching to an experimental native jna-based implementation
pixelFormat: 140203200597728
openGLContext: 140203226417888
contentView: 140203200489312
NSDefaultRunLoopMode: native@0x7fff849d1540
[timerWithTimeInterval:invocation:repeats:, timerWithFireDate:target:selector:userInfo:, scheduledTimerWithTimeInterval:repeats:block:, timerWithTimeInterval:repeats:block:, allocWithZone:, timerWithTimeInterval:target:selector:userInfo:repeats:, scheduledTimerWithTimeInterval:target:selector:userInfo:repeats:, scheduledTimerWithTimeInterval:invocation:repeats:]
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007fff61e56415, pid=919, tid=775
#
# JRE version: OpenJDK Runtime Environment (11.0.4+10) (build 11.0.4+10-b304.77)
# Java VM: OpenJDK 64-Bit Server VM (11.0.4+10-b304.77, mixed mode, tiered, compressed oops, g1 gc, bsd-amd64)
# Problematic frame:
# C  [libobjc.A.dylib+0xb415]  class_conformsToProtocol+0x138
#
# No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /Users/nicolaiemig/dev/engineEmi/hs_err_pid919.log
#
# If you would like to submit a bug report, please visit:
#   https://youtrack.jetbrains.com
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
```

Java crash Log on [pastebin](https://pastebin.com/3ALcJctd) because its huge

Maybe newer jogl, jna libraries available which fix that?

**Comment by soywiz** [comment]
Fixed on 1.4.3c. Now 1.4.3c uses JNA on all the targets.
Thanks both @emign  @viteinfinite for your help and patience!

**Comment by soywiz** [comment]
Forgot to link the fix to this issue for traceability sake: https://github.com/korlibs/korgw/commit/85815a457f101bfa796fec9b4b6c5b3f73712e87
